etcdctl/READMEv3.md

```diff
@@ -370,6 +370,64 @@ Simple reply
 [mirror]: ./doc/mirror_maker.md
 
 
+### SNAPSHOT [subcommand]
+
+SNAPSHOT takes a point-in-time snapshot of etcd and restores it into a fresh cluster.
+
+### SNAPSHOT SAVE \<filename\>
+
+SNAPSHOT SAVE writes a point-in-time snapshot of the etcd backend database to a file.
+
+#### Return value
+
+- On success, the backend snapshot is written to the given file path.
+
+- Error string if mirroring failed. Exit code is non-zero.
+
+#### Example
+
+Save a snapshot to "snapshot.db":
+```
+./etcdctl snapshot save snapshot.db
+```
+
+
+### SNAPSHOT RESTORE [options] \<filename\>
+
+SNAPSHOT RESTORE creates an etcd data directory for an etcd cluster member from a backend snapshot and a new cluster configuration. Restoring the snapshot into each node for a new cluster configuration will initialize a new etcd cluster preloaded by the snapshot data.
```

Comment:
a backend database snapshot file? (we call this backend database snapshot at line 379)